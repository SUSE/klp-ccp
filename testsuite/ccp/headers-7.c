void h(void)
{}

#include "headers-7.h"

void pu_f(void)
{
	g();
}
