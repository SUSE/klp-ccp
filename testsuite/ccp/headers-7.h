static void i(void)
{
	h();
}

static void g(void)
{}
