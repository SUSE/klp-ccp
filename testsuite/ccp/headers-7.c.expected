/* klp-ccp: from testsuite/ccp/headers-7.c */
void h(void);

#include "headers-7.h"

void klpp_pu_f(void)
{
	g();
}
